lib/recurly/base.php

```diff
@@ -133,10 +142,22 @@ protected function __valuesString() {
     return implode($values, ', ');
   }
 
+  public function getHref() {
+    return $this->_href;
+  }
+  // TODO: not sure this should be public
+  public function setHref($href) {
```

Comment:
It seems to me like it should be private. It's used here https://github.com/recurly/recurly-client-php/blob/51a876606d537ce7190392b82661dca259b86b90/lib/recurly/base.php#L356

I'm not sure why the href must be explicitly set like this. Do you have an idea?